app/src/main/java/net/bible/android/view/activity/navigation/History.kt

```diff
@@ -96,6 +108,6 @@ class History : ListActivityBase() {
 
     companion object {
         private const val TAG = "History"
-        private const val LIST_ITEM_TYPE = android.R.layout.simple_list_item_1
+        private const val LIST_ITEM_TYPE = R.layout.simple_list_item_2_modified
```

Comment:
You could name this history_list_item.xml

Opinions might vary but I'm not sure if we really need to keep these too generic.